src/sentry/web/frontend/debug/mail.py

```diff
@@ -155,6 +159,46 @@ def render(self, request: Request):
         )
 
 
+@method_decorator(csrf_exempt, name="dispatch")
+class MailPreviewView(View, abc.ABC):
```

Comment:
@Neo-Zhixing How is this abstraction going to work with Notification driven previews? Want to make sure this abstraction works for those kind of previews as well.

Reply:
It works exactly the same. To show this I've converted another view `DebugReleaseSummaryEmailView` which is a notification.